test/core/handshake/server_ssl.c

```diff
@@ -140,9 +140,11 @@ static bool server_ssl_test(const char *alpn_list[], unsigned int alpn_list_len,
   SSL_load_error_strings();
   OpenSSL_add_ssl_algorithms();
 
-  const SSL_METHOD *method = TLSv1_2_client_method();
+  const SSL_METHOD *method = TLS_client_method();
   SSL_CTX *ctx = SSL_CTX_new(method);
-  if (!ctx) {
+  if (!ctx ||
```

Comment:
The !SSL_CTX_set_min .... idiom is used in three places:  
   1.  test/.../server_ssl.c
   2.  test/.../client_ssl.c
   3.  src/core/tsi/ssl_transport_security.c

The options are either to include the convenience function in a header accessible by all three or to use the raw idiom in all three places.

Overall it appears that using the raw idiom is better/simpler for the time being.